authorizations.md

```diff
@@ -0,0 +1,93 @@
+# Authorizations Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+An authorizations is the representation of some rights that are available to a specific user (eperson) on a defined object, eventually the whole repository (site object).
+All the authorizations are always explicitly listed regardless to how they are grant, by direct policies via groups' membership or open to everyone (anonymous users).
+Please note that these endpoints are all READ-ONLY, authorizations are granted and withdrawn as result of business processes, configurations and resource policies.
+
+## Main Endpoint
+**/api/authz/authorizations**
+
+As we don't have yet an use case to iterate over all the existent authorizations the main endpoint is not implemented and a 405 error code is returned according to our [general error response codes](README.md#Error-codes).
+
+## Single Authorization
+**/api/authz/authorizations/<:id>**
+
+Provide detailed information about a specific authorization. The JSON response document is as follow
+
+```json
+{
+  "id": [eperson-uuid_]feature-id_object-uuid,
```

Comment:
Will it have any semantic associated with? Meaning should I parse the ID to get each part? I'm in favor of not having semantics here.

Reply:
The client doesn't really need to guess the authorization ID using the above algorithm. Search methods are provided to allow the client to check authorization depending on the object and the requested feature. Anyway, I have preferred to make the ID "syntax" explicit as we don't want to introduce new tables/persistence mechanism for the authorization but instead, as said at the top of the contract and in the wiki authorization are the result of processing resourcepolicies, configurations and business rules

Reply:
I'm OK with having the `id` of an authorization be a combined set of UUIDs.  I agree that the only way to more easily "assign" an ID here would be to store this info in a new database table or similar (which I'd rather not do for now -- that could always come later in v8 if we want to enhance the authorization system)

Reply:
I can see a conflict between https://github.com/DSpace/Rest7Contract/pull/92#discussion_r348985479 and this id.

If you have a need for supporting integer IDs, they will not be unique. You can have two authorizations related to a different type of object, with the same id using this syntax. I think to be unique, the type will need to be included in the id as well

Reply:
you are right @benbosman I will add the object type in the ID. It is not strictly needed when the object id is an uuid but it will help in all the case to speedup the lookup